src/js/lib/api.js

```diff
@@ -350,6 +350,61 @@ function getIssues(assignee = 'none', labels = []) {
         .then(formatIssueResults);
 }
 
+function getPreviousInstancesOfIssue(titleParts) {
```

Comment:
NAB - pretty self explanatory, but maybe useful to note that this is mainly used by accounting team for now? 🤷 

```suggestion
/**
 * Get previous instance of issue - used by accounting team to find the previous issue for the currently open chore
 *
 * @param {String[]} titleParts
 * @returns {Promise}
 */
function getPreviousInstancesOfIssue(titleParts) {
```